### PR TITLE
Add documentation for using environment variables buildpack

### LINKED ISSUE #88: The site does not have documentation of how to use the Paketo Environment Variables Buildpack
## What happened?

Please provide some details about the task you are trying to accomplish and
what went wrong.

* What were you attempting to do?
I was trying to learn how to use the [Environment Variables buildpack](https://github.com/paketo-buildpacks/environment-variables) by reading the [Configuring Paketo Buildpacks](https://paketo.io/docs/buildpacks/configuration/) section of the site.

* What did you expect to happen?
I expected to find some docs on the site

* What was the actual behavior? Please provide log output, if possible.
There were none for this buildpack.

## Checklist

Please confirm the following:

* [ ] I have included log output.
* [ ] The log output includes an error message.
* [ ] I have included steps for reproduction.

### PR DESCRIPTION
**Pull request by dmikusa-pivotal**
## Summary

Adds documentation for using the environment variables buildpack.

## Use Cases

Resolves #88 

## Checklist

* [x] I have viewed, signed, and submitted the Contributor License Agreement.
* [x] I have linked issue(s) that this PR should close using keywords or the Github UI (See [docs](https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue))
* [n/a] I have added an integration test, if necessary.
* [x] I have reviewed the [styleguide](https://github.com/paketo-buildpacks/community/blob/main/STYLEGUIDE.md) for guidance on my code quality.

### CODE REVIEW

**Review by ForestEckhardt — APPROVED** [review]

**Comment by ForestEckhardt** [comment]
@kvedurmu I would love for you to take a look at this as well

File: content/docs/buildpacks/configuration.md [hunk 1]
```diff
@@ -65,6 +71,26 @@ curl -s http://localhost:8082/actuator/health
 
 Programs invoked at runtime, including the application itself, will accept environment as they would normally.
 
+#### Image Embedded Environment Variables
+
+Users may embedded environment variables into the images created by using the [Environment Variables buildpack][bp/environment-variables]. The Environment Variables buildpack looks for environment variables matching the pattern `$BPE_*`. When detected, the buildpack will modify the launch environment to adjust the specified variables. This is a good way to set non-sensitive configuration values such as defaults or modifiy environment variables that you do not want your image users to need to set.
```
Comment (kvedurmu) [inline content/docs/buildpacks/configuration.md:76]:
```suggestion
Users may embed environment variables into the images created by using the [Environment Variables buildpack][bp/environment-variables]. The Environment Variables buildpack looks for environment variables matching the pattern `$BPE_*`. When detected, the buildpack will modify the launch environment to adjust the specified variables. This is a good way to set non-sensitive configuration values such as defaults or modify environment variables that you do not need users to set.
```

**Review by kvedurmu — CHANGES_REQUESTED** [review]
Looks great overall. Just found a few typos

**Comment by dmikusa-pivotal** [comment]
@kvedurmu - Looks good, I committed your suggestion.

**Review by kvedurmu — APPROVED** [review]

**Pull request merged**